src/lualib/Symbol.ts

```diff
@@ -1,11 +1,6 @@
-// tslint:disable-next-line: variable-name
 const ____symbolMetatable = {
-    __tostring(): string {
-        if (this.description === undefined) {
-            return "Symbol()";
-        } else {
-            return "Symbol(" + this.description + ")";
-        }
+    __tostring(this: symbol): string {
+        return `Symbol(${this.description || ""})`;
```

Comment:
Wouldn't `??` be more correct here?

Reply:
We don't support nullish coalescing yet, but in general it doesn't matter since the only possible falsy value there is an empty string